armtemplates/template.bicep

```
@@ -0,0 +1,280 @@
+param virtualMachines_iagora_name string = 'iagora'
+param publicIPAddresses_iagora_ip_name string = 'iagora-ip'
+param virtualNetworks_iagora_vnet_name string = 'iagora-vnet'
+param networkInterfaces_iagora731_z1_name string = 'iagora731_z1'
+param networkSecurityGroups_iagora_nsg_name string = 'iagora-nsg'
+param sshPublicKey string 
+param vmSize string = 'Standard_B2s' 
+
+//NSG ressourc
+resource networkSecurityGroups_iagora_nsg_name_resource 'Microsoft.Network/networkSecurityGroups@2023-11-01' = {
```

```suggestion
resource networkSecurityGroups 'Microsoft.Network/networkSecurityGroups@2023-11-01' = {
```
Could we use shorter camelCased names for the symbolic resource references?

---

Could you shorten other resource references as well?

For example, the following:
```bicep
resource networkInterfaces_iagora731_z1_name_resource ...
resource virtualNetworks_iagora_vnet_name_default ...
```
becomes:
```bicep
resource networkInterfaces ...
resource virtualNetworks ...
```